miner/worker.go

```diff
@@ -835,34 +854,29 @@ func (w *worker) commitTransaction(tx *types.Transaction, coinbase common.Addres
 		StorageProofs: make(map[string]map[string][]hexutil.Bytes),
 	}
 
-	receipt, err := core.ApplyTransaction(w.chainConfig, w.chain, &coinbase, w.current.gasPool, w.current.state, w.current.header, tx, &w.current.header.GasUsed, *w.chain.GetVMConfig())
-	if err != nil {
-		w.current.state.RevertToSnapshot(snap)
-		return nil, err
-	}
-
 	proofAccounts := tracer.UpdatedAccounts()
 	if w.chainConfig.Scroll.FeeVaultEnabled() {
 		proofAccounts[*w.chainConfig.Scroll.FeeVaultAddress] = struct{}{}
 	} else {
 		proofAccounts[coinbase] = struct{}{}
 	}
 	proofAccounts[rcfg.L1GasPriceOracleAddress] = struct{}{}
+	proofAccounts[coinbase] = struct{}{}
```

Comment:
do we still need this if we have L857~L862

Reply:
No need.